mergify_engine/redis_utils.py

```diff
@@ -67,12 +67,13 @@ def register_script(script: str) -> ScriptIdT:
 # it works but if a script is loaded into two redis, this won't works as expected
 # as the app will think it's already loaded while it's not...
 async def load_script(
-    redis: typing.Union[RedisCache, RedisStream], script_id: ScriptIdT
+    redis: "redispy.connection.Connection", script_id: ScriptIdT
 ) -> None:
     global SCRIPTS
     sha, script = SCRIPTS[script_id]
-    # FIXME(sileht): weird, this method is typed on redis-py
-    newsha = await redis.script_load(script)  # type: ignore[no-untyped-call]
+    await redis.send_command("SCRIPT LOAD", script)
```

Comment:
what's the rational for not using `script_load` anymore?

Reply:
because when using `redis_connect_func` parameter, what is passed as an argument to the function is a redis `Connection` object and not a redis `Client`. So script_load is not defined for a `Connection` object, instead, we leverage to send direct commands through the `Connection`